extensions/classifiers/WinnowingPreprocessingService.ts

```diff
@@ -59,7 +59,7 @@ oppia.factory('WinnowingPreprocessingService', [function() {
       }
 
       var fingerprint = [];
-      fingerprintHashesIndex.forEach(function(hashIndex) {
+      fingerprintHashesIndex.forEach(function(hashIndex: number) {
```

Comment:
Remember to revert this when your other PR goes in!

Reply:
Sure.